certified-connectors/intelliHR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7967 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "V1",
+    "title": "intelliHR",
+    "description": "You can find developer's guide and more documentation on [https://developers.intellihr.io](https://developers.intellihr.io)\n\n# Authentication\n\n<!-- ReDoc-Inject: <security-definitions> -->",
```

Comment:
@wolskis Can you please provide a brief about your connector in the description value as we expect it should describe concisely the main purpose and value offered by your connector.

Instructions - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#description)

Reply:
Amended field